.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: python
+python:
+  - "3.8"
+#create requirements.txt file
+before_install:
+  - pip freeze --local > requirements.txt
```

Comment:
can we get rid of this line along with the before_install, since we already have a requirements.txt file in the root

Reply:
Done.